set.mm

```diff
@@ -112439,14 +112442,19 @@ this axiom (with the defined operation in place of ` x. ` ) follows as a
       GWHUUGMJWIWJWKWLWMVM $.
   $}
 
-  $( ` 0 ` is a real number.  See also ~ 0reALT .  (Contributed by Eric
-     Schmidt, 21-May-2007.)  (Revised by Scott Fenton, 3-Jan-2013.) $)
+  $( The number 1 is real, deduction form.  (Contributed by David A. Wheeler,
+     6-Dec-2018.) $)
+  1red $p |- ( ph -> 1 e. RR ) $=
+    ( c1 cr wcel 1re a1i ) BCDAEF $.
+
+  $( The number 0 is real.  See also ~ 0reALT .  (Contributed by Eric Schmidt,
+     21-May-2007.)  (Revised by Scott Fenton, 3-Jan-2013.) $)
   0re $p |- 0 e. RR $=
     ( vx c1 cr wcel cv caddc cc0 wceq wrex 1re ax-rnegex readdcl mpan syl5ibcom
     co eleq1 rexlimiv mp2b ) BCDZBAEZFOZGHZACIGCDZJABKUBUCACTCDZUACDZUBUCSUDUEJ
     BTLMUAGCPNQR $.
 
-  $( ` 0 ` is a real number, deductive form.  (Contributed by David A. Wheeler,
+  $( The number 0 is real, deduction form.  (Contributed by David A. Wheeler,
```

Comment:
Since in the following ".. is complex." is replaced by ".. is a complex number.", we should write "The number 0 is a real number" here (and in other comments).

Reply:
In the comments I did modify, I used:
* "is a complex number" instead of "is complex"
* "number xxx is real" instead of "number xxx is a real number"

("number" is already in the subjet, and the adjective "real" for this meaning is common, contrary to the adjective "complex").

Reply:
@avekens : as for capitalization, maybe we could harmonize at some point, although usage varies. The English language likes capitalization: even when a word derived from a person becomes a common adjective, it generally keeps its initial capital letter, hence "Cartesian" is probably more proper. On the other hand, if it becomes "super common", then it loses its capital initial, like "abelian" !

Reply:
> ("number" is already in the subject, and the adjective "real" for this meaning is common, contrary to the adjective "complex").

We still have many comment in the form "The number .. is a complex number" with the word "number repeated. I do not know if ".. is real" is more common than ".. is complex" in the English literature, in German there seem to be no difference.

But this is only a minor thing, we can stop tis discussion now.